### ISSUE TITLE
Install instructions are missing

### ISSUE THREAD

**Issue by Riduidel** [andirady/pom-cli#11]
As far as I understand, there is no documented way to install a *packaged* version of pom-cli. Am I missing something?

**Comment by andirady** [comment]
@Riduidel, yes, it's still missing. I'm still figuring out how to package this tool. 😅

**Comment by Riduidel** [comment]
What I've done no longer than today is create a GitHub action which, when the release is created, fires a build (https://github.com/Riduidel/aadarchi-technology-detector/blob/main/.github/workflows/on_github_release_publish_maven_release.yaml). You could use the same mechanism to create the native executable for various platforms (through matrix build) then attach the generated executables through release (I think the good action for that is https://github.com/marketplace/actions/upload-files-to-a-github-release).
I will try to take some time to work on that. At worst, I can contribute that during hacktoberfest.

**Comment by andirady** [comment]
@Riduidel, there's also [Jreleaser](https://jreleaser.org/) and they have sample in https://github.com/jreleaser/helloworld-java-graalvm.
My issue with the example is it downloads the JDKs.
